translation/languages/ro.csv

```diff
@@ -0,0 +1,99 @@
+View Forum,Vizualizează forum
+Why <span>BitcoinZ?</span>,De ce <span>BitcoinZ?</span>
+We are a fully decentralized network,Suntem o reţea 100% descentralizată
+Here are some advantages,Avantaje
+Anonymity,Anonimitate
+All transactions are completely anonymous.,Toate tranzacţiile sunt complet anonime. 
+Support for different platforms,Support pentru platforme diferite. 
+"Our wallet is compatible with Linux, Windows, iOS and Android.","Portofelul nostru este compatibil cu Linux, Windows, iOS si Android. "
+Community,Comunitate
+"We listen to our dear community, so we can succeed.","Pentru a reuţi, ascultam de comunitatea noastra dragă. "
+Active price growth,Creştere activa a preţurilor
+BitcoinZ is focused on organic growth.,BitcoinY se axează pe creştere organică.
+Exchanges,Exchange-uri
+Buy or sell your BitcoinZ coins,Cumpără sau vinde-ţi monedele BitcoinZ
+Download <span>Wallet</span>,Downloadează <span>portofelul</span>
+"Linux, Windows and mobile wallets","Linux, Windows şi portofele pentru mobil. "
+Road <span>Map</span>,<span>Ţeluri </span>
+BitcoinZ Core Client,BitcoinZ Core Client
+Block Explorer,Block Explorer
+Mining Pools,Pool-uri de minat
+GUI Wallet,Poftofel cu interfaţă grafică
+iOS &amp; Android Wallet,Portofel pentru iOS & Android
+Commerce Plugins,Plug-in-uri pentru comerţ
+Exchange Listing,Listări pe exchange-uri
+Masquerade Technology,Tehnologie mascată
+Decentralization Packages,Pachete descentralizate
+Game Development,Dezolvate pentru jocuri 
+zk-SNARKs From Mobile,zk-SNARKs pentru mobil 
+Public Relations,Relaţii publice
+why choose <span>BitcoinZ</span>,De ce să alegi <span>BitcoinZ</span>
+Here are a few <span>reasons</span>,Iată nişte <span>motive</span>
+WARNING: Always remember about the pitfalls and keep your backup keys in a safe place!,ATENŢIE: Ţine cont de capcane şi ţine-ţi cheile de back-up într-un loc sigur!
+Decentralized <span>Mining</span>,<span>Minat<span> descentralizat
+As a miner you should not use the biggest pools to follow main principles.<br>Easy to mine<br>Equihash algorithm.<br>Decentralized Exchanges,Ca miner nu trebuie să foloseşti cele mai mari pool-uri pentru a urmări principiile. <br>Uşor de minat<br>Algoritm Equihash<br> Exchange-uri descentralizate
+No <span>Pre-Mine</span>,Fără <span>pre-minare</span>
+Pure coin. No pre-mine.,Monedă pură. Fără pre-minare
+21 billion coins! Everyone in the world can have at least one BTCZ!,21 de miliarde de monede! Toată lumea poate deţine măcar 1 BTCZ!
+Our community is actively developing and growing every day.,Comunitatea noastră dezvoltă şi creşte în fiecare zi. 

```

Comment:
there should be a "se" between "noastră" and "dezvoltă" and become "noastră se dezvoltă"